doc/neomake.txt

```diff
@@ -720,14 +721,32 @@ You can use the following options:
     be useful for non-current windows.
     Default: `'%s'` (used with |printf()|).
 
-  - format_status: used to wrap the whole status (if Neomake is not enabled).
+  - format_status: used to wrap the whole status.
+    This is a |printf()| format string, where `%s` gets replaced with the
+    whole status (and any literal/non-printf `%` needs to be escaped as `%%`).
+    Default: not used / `'%s'`
+  - format_status_disabled: used to wrap the whole status when disabled.
     This is a |printf()| format string, where `%s` gets replaced with the
     whole status (and any literal/non-printf `%` needs to be escaped as `%%`).
+    Default: `'{{disabled_info}} %s'`
+  - format_status_enabled: used to wrap the whole status when enabled.
+    This is a |printf()| format string, where `%s` gets replaced with the
+    whole status (and any literal/non-printf `%` needs to be escaped as `%%`).
+    Default: not used / `'%s'`
+  - format_running_job_file: Wrap the running job name for file-level makers.
     Default: not used / `'%s'`
+  - format_running_job_project: Wrap the running job name for project makers.
+    Default: `'%s!'`.
+  - running_jobs_separator: Separator for formatted running job names.
+    Default: `', '`.
 
 You can use the following format placeholders:
   - `{{running_job_names}}}`: comma-separated list of running jobs (typically
     their maker names)
+  - `{{scope}}`: When manually overriden, the scope of the disable. One of
```

Comment:
s/the disable/the disabling/
But it also is made available for `format_status_enabled`.
I think we should rename it to `disabled_scope` and use it only when disabled for now.
If somebody really wants it for `format_status_enabled` we could add it as `enabled_scope` then.

Reply:
format_disabled_info and format_enabled_info is not documented yet.
Pushed some fixup, which also removes format_enabled_info.
What do you think?

Reply:
LGTM, I only added the "enabled" version in case anyone wants the old coded behaviour of showing `b+` if "disabled" is not -1. Personally I wouldn't use it.